### DISCUSSION TITLE
How and When to Create a URL multimedia record

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#325] [HOW TO...]
**Used By**: All
**In Use?**: Yes

See recommendations here: 
- https://github.com/fieldmuseum/EMu-Documentation/discussions/426
- Axiell's ["How to add a non-file resource (URL and reference)"](https://help.emu.axiell.com/latest/en/Topics/Common/How%20to%20add,%20edit%20and%20save%20multimedia.htm)

[Googledoc Workflow](https://drive.google.com/drive/u/0/search?q=EMu_batch_add_URL_links.pdf)

**Comment by magpiedin** [comment]
(corresponding FAQ: https://github.com/fieldmuseum/EMu-Documentation/discussions/426)
